pallets/subtensor/src/lib.rs

```diff
@@ -1825,10 +1825,20 @@ where
                 priority: Self::get_priority_vanilla(),
                 ..Default::default()
             }),
-            Some(Call::register { .. }) => Ok(ValidTransaction {
-                priority: Self::get_priority_vanilla(),
-                ..Default::default()
-            }),
+            Some(Call::register { netuid, .. }) => {
```

Comment:
can we also add this for `burned_register` ? We should be able to use `||` in the method